yarn.lock

```diff
@@ -42,11 +42,12 @@
     minimatch "^3.0.4"
 
 "@actions/http-client@^2.0.1", "@actions/http-client@^2.1.1":
-  version "2.1.1"
-  resolved "https://registry.yarnpkg.com/@actions/http-client/-/http-client-2.1.1.tgz#a8e97699c315bed0ecaeaaeb640948470d4586a0"
-  integrity sha512-qhrkRMB40bbbLo7gF+0vu+X+UawOvQQqNAA/5Unx774RS8poaOhThDOG6BGmxvAnxhQnDp2BG/ZUm65xZILTpw==
+  version "2.2.0"
```

Comment:
Could you undo all changes in yarn.lock?
They can cause failures, etc and updating usually makes it worse more than it helps from past experience.
In any case updating it should be done in its own commit/PR so easy to revert if any issue.

Reply:
I'll split the yarn.lock changes.